OpenMarket/OpenMarket/Extension/Double+.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Double+.swift
```

Comment:
extension 파일명은 타입명+extension.swift로 작성하는 편입니다.
```suggestion
//  Double+extension.swift
```

Reply:
수정하였습니다. :)